.vscode/settings.json

```diff
@@ -5,6 +5,6 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[typescriptreact]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
did you mean to commit this? i think we should be git ignoring these file changes